src/SFA.DAS.EmployerIncentives.Data/ApprenticeshipIncentives/ApprenticeshipIncentiveDataRepository.cs

```diff
@@ -183,6 +185,25 @@ private void UpdateApprenticeshipIncentive(ApprenticeshipIncentive updatedIncent
                     _dbContext.BreakInLearnings.Add(breakInLearning);
                 }
             }
+
+            foreach (var employmentCheck in updatedIncentive.EmploymentChecks)
+            {
+                var existingEmploymentCheck = existingIncentive.EmploymentChecks.SingleOrDefault(p => p.Id == employmentCheck.Id);
+
+                if (existingEmploymentCheck != null)
+                {
+                    _dbContext.Entry(existingEmploymentCheck).CurrentValues.SetValues(employmentCheck);
+                    if (_dbContext.Entry(existingEmploymentCheck).State == EntityState.Modified)
+                    {
+                        existingEmploymentCheck.ResultDateTime = DateTime.Now;
```

Comment:
This whole block should be removed.  The ResultDateTime is provided when the Employment check is updated, the repository shouldn't be changing business data.